doc/conf.py

```diff
@@ -16,6 +16,20 @@
 import sys
 import os
 
+currentdir = os.path.abspath(os.path.dirname(__file__))
+sys.path.append(os.path.join(currentdir, 'tools'))
+import buildmodref
+
+# autogenerate api documentation
+# (see https://github.com/rtfd/readthedocs.org/issues/1139)
+def generateapidoc(_):
```

Comment:
Would this work if this function had no input? 

    def generateapidoc():

Instead of what we currently have?